lib/cdo/analytics/milestone_parser.rb

```diff
@@ -20,8 +20,6 @@ class MilestoneParser
     production-daemon
     staging
     test
-    levelbuilder-staging
-    levelbuilder-development
```

Comment:
I don't recognize `levelbuilder-development`; does anyone know what this is or why it was included? I couldn't find any other reference to it in our codebase or in EC2, so figured it would be safe to remove here. Please let me know if you have any reason to suspect otherwise!